---
title: "Dr. Mangul featured in COVID-19 preparedness article on GenomeWeb"

author: Dottie Yu
member-page: yu-dottie.html


---

Dr. Mangul was featured in a recent GenomeWeb article that sheds light on what we can learn from the bioinformatics response to the COVID-19 pandemic, to prepare for future pandemics. 
 
According to Dr. Mangul, viral sequencing was considered a niche field before COVID-19. Some platforms existed, like Chan Zuckerburg ID (CZ ID) and Nextstrain, which were rapidly repurposed for pandemic during its early days. The response to the pandemic was swift and bold—but in reflecting, Dr. Mangul concludes that “retrospectively, we can do a better job, and I think we should.” He explains that the metadata stored with samples was often incomplete, missing useful information such as when the sample was collected and the severity of the individual’s infection. He elaborates the value of this metadata: “there’s so many interesting fundamental biological questions you can ask about, [like] the evolution of the virus [and] how it’s transmitted.”
 
Dr. Mangul published a related paper in Nature Methods on our bioinformatics response to COVID-19. The study finds that 80% of GISAID data is generated via Illumina short-read sequencing, although long-read sequencing (LRS) is supported by the database. He attributes this discrepancy to availability and “a matter of choice,” although he notes that LRS technologies have quickly advanced and are now comparable in accuracy with Illumina. Dr. Mangul highlights the potential of leveraging LRS technologies in pandemic response, explaining that “especially for viruses, both PacBio and Oxford Nanopore [LRS technologies] are very promising because they cover longer stretches of viral genomes.” He adds, “it would be helpful to see more benchmarking studies looking at the accuracy between sequencing manufacturers as well as the efficacy of various bioinformatics tools.”
 
To conclude, the COVID-19 pandemic has propelled bioinformatics into the spotlight as a crucial tool for understanding and combating infectious diseases. "Some choices that we made during the pandemic were not informative and not data-driven because we didn't have the data or we were in a rush to implement things," Mangul explains. "There is so much to learn from that and improve our bioinformatics, improve sequencing, and improve experimental protocols so that we are ready for the next public health emergency. “ By leveraging these lessons, the bioinformatics community can contribute to mitigating the impact of future outbreaks and ensuring a more effective and coordinated response.
 
[Read the full article](https://www.genomeweb.com/informatics/covid-19-emergency-ends-bioinformaticians-look-back-inform-future-pandemic-preparedness#.ZGfyrezMLX3)
