<!DOCTYPE html>
<html>

  <head>
	<meta charset="utf-8">
	<meta http-equiv="X-UA-Compatible" content="IE=edge">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<title>Stephan Sanders</title>
	<meta name="description" content="">
	<link rel="stylesheet" href="http://localhost:4000/css/main.css">
	<link rel="canonical" href="http://localhost:4000/team/stephansanders.html">
	<link rel="shortcut icon" type ="image/x-icon" href="http://localhost:4000/images/favicon.ico">
	<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.13/css/all.css" integrity="sha384-DNOHZ68U8hZfKXOrtjWvjxusGo9WQnrNx2sqG0tfsghAvtVlRW3tvkXWZh58N9jp" crossorigin="anonymous">
	<link rel="stylesheet" href="/css/academicons/css/academicons.css">

	
</head>


  <body>

    <div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container-fluid">
	<div class="navbar-header">
	  <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar-collapse-1" aria-expanded="false">
		<span class="sr-only">Toggle navigation</span>
		<span class="icon-bar"></span>
		<span class="icon-bar"></span>
		<span class="icon-bar"></span>
	  </button>
	
    <a class="navbar-brand" href="http://localhost:4000/">Sanders Lab @ UCSF</a>
	</div>

	<div class="collapse navbar-collapse" id="navbar-collapse-1">
	  <ul class="nav navbar-nav navbar-right">
		<li><a href="http://localhost:4000/">Home</a></li>
		<li><a href="http://localhost:4000/team">Team</a></li>
		<li><a href="http://localhost:4000/research">Research</a></li>
		<li><a href="http://localhost:4000/publications">Publications</a></li>
		<li><a href="http://localhost:4000/code">Codes</a></li>
		<li><a href="http://localhost:4000/contact">Contact</a></li>
	  </ul>
	</div>
	
  </div>
</div>


    <div class="container-fluid">
      <div class="row">
        

<div class="row">
	<div class="col-sm-12">

			<img class="pull-left pad-right content-photo" width=160 src="http://localhost:4000/images/teampic/stephansanders.jpg">


				<div class="bigtitle">
					Stephan Sanders
				</div>
				<p><div class="smallhead">
					Principal Investigator
				</div></p>
	</div>
</div>

<div class="bigspacer"></div>

<div class="row">
	<div class="col-sm-3">

		
		<div class="smallhead">
			UCSF Profile
		</div>
		<div class="pad-left note">
			<div class="smallspacer"></div>
			<i class="fa fa-university"></i>
			<a class="off" href="https://profiles.ucsf.edu/stephan.sanders">stephan.sanders</a>
		</div>
		<div class="bigspacer"></div>
		

		

		
		<div class="smallhead">
			GitHub
		</div>
		<div class="pad-left note">
			<div class="smallspacer"></div>
			<i class="fab fa-github"></i>
			<a class="off" href="https://github.com/sanderslab">sanderslab</a>
		</div>
		<div class="bigspacer"></div>
		

		
		<div class="smallhead">
			Google Scholar
		</div>
		<div class="pad-left note">
			<div class="smallspacer"></div>
			<i class="ai ai-google-scholar"></i>
			<a class="off" href="http://scholar.google.com/citations?user=O1EuSPYAAAAJ">Stephan Sanders</a>
		</div>
		<div class="bigspacer"></div>
		

		

		

	</div>
	<div class="col-sm-6" style="text-align:justify">
		<i class="content" style="margin:auto"><h3 id="overview">Overview</h3>
<p>Dr. Sanders trained as a pediatric physician at Nottingham and London in the UK before undertaking a PhD and postdoctoral research position at Yale. He is now an Assistant Professor at UCSF in the Department of Psychiatry. His research focuses on using genomics and bioinformatics to understand the etiology of human diseases and disorders, especially ASD, with a special interest in the mechanisms of sex bias.</p>

<p>Using SNP genotyping arrays on samples in the Simons Simplex Collection (SSC), he worked with the SSC Genomic Consortium (SSCGC) to clarify the important role of de novo copy number variation (CNVs) in ASD, which included the discovery of duplications at 7q11.23; this finding is of note since deletions at the same locus cause Williams-Beuren syndrome, which is associated with increased sociability (Sanders et al. Neuron 2011). In the same sample cohort, he used exome sequencing to show that de novo loss of function (LoF) mutations are associated with ASD and to establish a statistical framework for identifying the specific genes involved in ASD pathology based on these de novo events. This statistical method was used to identify the ASD risk genes SCN2A and KATNAL2 (Sanders et al. Nature 2012).</p>

<p>These methods have been further developed and applied to larger ASD cohorts. Dr. Sanders was a co-first author of the exome analysis of the entire SSC (Iossifov et al. Nature 2014) and was also a co-author of the Autism Sequencing Consortium (ASC) exome analysis (De Rubeis et al. Nature 2014). Recently, Dr. Sanders integrated CNV data from the entire SSC and Autism Genome Project (Pinto et al. AJHG 2014) with exome data from the SSC and ASC. In the largest genomic analysis in ASD to date 65 ASD risk genes and 6 ASD risk CNV loci were identified by the SSCGC (Sanders et al. Neuron 2015). In addition, by comparing the CNV and exome data this analysis showed that a single critical gene is often present in small de novo deletions, where as large de novo CNVs appear to contain multiple ASD risk genes of low effect.</p>

<p>Dr. Sanders has also worked in collaboration with geneticists, statisticians and neuroscientists to investigate what these ASD genes can reveal about ASD neuropathology. By integrating spatiotemporal gene expression data from the human brain (BrainSpan) and the results of exome sequencing, the group built and interrogated spatiotemporal co-expression networks to identify points of convergence in gene expression. This approach has highlighted the role of layer 5/6 glutamatergic neurons in the frontal cortex in mid-fetal development in the causation of ASD (Willsey et al. Cell 2013).</p>

<p>Throughout these analyses Dr. Sanders has sought to identify the reason behind the 4:1 excess of ASD diagnoses in males. His work has been critical in supporting the female protective effect hypothesis through the identification of a higher burden of ASD risk factors in females, specifically in de novo CNVs (Sanders et al. Neuron 2011, Sanders et al. Neuron 2015), de novo frameshift mutations (Dong et al. Cell Reports 2014), and de novo LoF mutations (Iossifov et al. Nature 2014, De Rubeis et al. Nature 2014). In addition, he has demonstrated that de novo mutations target the same genes in males and females (Sanders et al. Neuron 2015), further supporting a protective effect in females. Finally, through a female only GWAS analysis his lab showed that there was no single common variant responsible for such a protective effect in a subset of females (Gockley et al. Molecular Autism 2015). His lab continues to work on the nature of this female protection through genomic analysis.</p>

<h3 id="education">Education</h3>
<ul>
  <li>PhD, Genetics, Yale University (2014)</li>
  <li>Postdoctoral Research Associate, State Lab, Yale University (2007-2011)</li>
  <li>Membership of the Royal College of Paediatrics and Child Health, UK - equivalent to board exams in USA (2007)</li>
  <li>Bachelor of Medicine; Bachelor of Surgery (BMBS - equivalent to MD in USA); Nottingham University Medical School, UK (2003)</li>
</ul>
</i>

	</div>

</div>

      </div>
    </div>

    <div id="footer" class="panel">
  <div class="panel-footer">
	<div class="container-fluid">
	  <div class="row">


			<ul class="nav navbar-nav navbar-right">
				<li><a href="https://psych.ucsf.edu/">Department of Psychiatry</a></li>
				<li><a href="https://www.ucsf.edu/">UCSF</a></li>
				<li><a href="http://localhost:4000/about">About</a></li>
				<li><a href="http://localhost:4000/recruitment">Recruitment</a></li>
			</ul>

		
	  </div>
	</div>
  </div>
</div>

<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
<script src="http://localhost:4000/js/bootstrap.min.js"></script>


  </body>

</html>
